lib/mail_interceptor/initializers/zerobounce.rb

```diff
@@ -0,0 +1,6 @@
+require "zerobounce"
+
+Zerobounce.configure do |config|
+  config.apikey = ENV['ZEROBOUNCE_API_KEY']
```

Comment:
@mustafapc19 Can make this `api_key`?

Reply:
@ghousemohamed I looked into `Zerobounce` codebase they seem to be expecting and using `api_key`.